packages/faustwp-cli/src/telemetry/sendTelemetryData.ts

```diff
@@ -1,39 +1,23 @@
 import 'isomorphic-fetch';
 
+import { getWpUrl } from '../utils/getWpUrl.js';
+import { getWpSecret } from '../utils/getWpSecret.js';
 import { TelemetryData } from './marshallTelemetryData.js';
-
-const GA_TRACKING_ENDPOINT = 'https://www.google-analytics.com/mp/collect';
-const GA_TRACKING_ID = 'G-KPVSTHK1G4';
-const GA_API_SECRET = '-SLuZb8JTbWkWcT5BD032w';
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const WP_TELEMETRY_ENDPOINT = `${getWpUrl()!}/wp-json/faustwp/v1/process_telemetry`;
 
 /**
- * Send the marshalled telemetry data to GA.
+ * Send the marshalled telemetry data to Faust Telemetry API.
  *
- * @param payload The data being sent to GA
- * @param anonymousId The anonymous ID of the machine we captured during init
+ * @param payload The data being sent to telemetry API
  */
-export const sendTelemetryData = (
-  payload: TelemetryData,
-  anonymousId: string,
-) => {
-  const body = {
-    client_id: anonymousId,
-    events: [
-      {
-        name: 'telemetry_event',
-        params: payload,
-      },
-    ],
-  };
-
-  return fetch(
-    `${GA_TRACKING_ENDPOINT}?${new URLSearchParams({
-      api_secret: GA_API_SECRET,
-      measurement_id: GA_TRACKING_ID,
-    }).toString()}`,
-    {
-      method: 'POST',
-      body: JSON.stringify(body),
+export const sendTelemetryData = (payload: TelemetryData) => {
+  return fetch(WP_TELEMETRY_ENDPOINT, {
+    headers: {
+      'x-faustwp-secret': getWpSecret() || '',
+      'Content-Type': 'application/json',
```

Comment:
Instead of returning an empty string here, let's throw an error instead of `getWpSecret` is undefined, since we always want this telemetry data to be sent with the secret key

```suggestion
  if(!getWpSecret()) {
    throw new Error('Faust secret key is required');
  }
  
  return fetch(WP_TELEMETRY_ENDPOINT, {
    headers: {
      'x-faustwp-secret': getWpSecret(),
      'Content-Type': 'application/json',
```